cmd/utils/flags.go

```diff
@@ -1151,7 +1156,7 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 	if ctx.GlobalIsSet(NetworkIdFlag.Name) {
 		cfg.NetworkId = ctx.GlobalUint64(NetworkIdFlag.Name)
 	}
-
+	cfg.ExitWhenSynced = ctx.GlobalDuration(ExitWhenSyncedFlag.Name)
```

Comment:
fixed

Reply:
Besides, please also delete the ExitWhenSynced definition in the eth.config and revert the auto generated config file

Reply:
I will revert it, but I want to confirm that means making it like it was before this PR?  I am asking because updating the auto-generated config file was requested earlier?

Reply:
I mean `ExitWhenSynced ` is not used in the eth protocol, so we don't need to add this into the configuration. Thus we don't need to regenerate the config file.
I apologize that I asked you to auto-generated config file before is I didn't realize that we don't need the `ExitWhenSynced ` at all.

Reply:
No problem, fixed